recipes/r-prabclus/2.2_6/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+{% set package = 'prabclus' %}
+{% set version = '2.2-6' %}
+
+package:
+  name: "r-{{ package }}"
+  version: "{{ version | replace('-', '_') }}"
+
+source:
+  fn: "{{ package }}_{{ version | replace('-', '_') }}.tar.gz"
+  url:
+    - "https://cran.r-project.org/src/contrib/{{ package }}_{{ version }}.tar.gz"
+    - "http://cran.r-project.org/src/contrib/Archive/{{ package }}/{{ package }}_{{ version }}.tar.gz"
+  sha256: 41792980e40ba18204fab92d85120dcd468860e2204e52fb42636c6f7aee5a62
+
+build:
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base >=2.10
```

Comment:
I don't know if it helps. All I know is that it usually causes problems (not in general, but with all packages where conda pins automatically: r-base, python, perl).

After that, we can investigate why it wants to build against r-base 3.2.2. to make this clear, it usually doesn't. Something is wrong here.